### PR TITLE
Updating Moneo tag and ansible install

### PR DESCRIPTION
**Pull request by rafsalas19**
Updating Moneo tag and ansible install

### CODE REVIEW

File: ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install_utils.sh [hunk 1]
```diff
@@ -42,7 +42,6 @@ apt-get -y install numactl \
                    libnl-route-3-dev \
                    net-tools \
                    libsecret-1-0 \
-		   ansible \
```
Comment (jithinjosepkl) [inline ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install_utils.sh:45]: should we have this change to the LTS image also?

Comment (rafsalas19) [reply ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install_utils.sh:45]: Let me check. I am not sure if we released Moneo for LTS

Comment (rafsalas19) [reply ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install_utils.sh:45]: So the LTS version doesn't have Moneo. Pretty sure we decided not to include it.

**Review by jithinjosepkl — APPROVED** [review]

File: ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install_utils.sh (hunk 1 above)
Comment (moes1) [reply ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install_utils.sh:45]: Do we need ansible in the LTS version? We recently pulled all these apt installs into the ubuntu/common/install_utils.sh file, but if we need to specialize by image version again we can do that...

**Review by moes1 — APPROVED** [review]

**Pull request merged**